src/MyParcelComApi.php

```diff
@@ -505,11 +501,28 @@ protected function getResourcesPromise($uri, $ttl = self::TTL_10MIN)
             });
     }
 
+    protected function getResourceCollection($uri, $ttl = self::TTL_10MIN)
+    {
+        return new PromiseCollection(
+            function ($pageNumber, $pageSize) use ($uri, $ttl) {
+                $url = (new UrlBuilder($uri))->addQuery([
+                    'page[number]' => $pageNumber,
+                    'page[size]'   => $pageSize,
+                ])->getUrl();
+
+                return $this->doRequest($url, 'get', [], [], $ttl);
+            }, function ($data) {
+            return $this->jsonToResources($data);
+        });
+    }
+
     /**
      * {@inheritdoc}
      */
-    public function doRequest($uri, $method = 'get', array $body = [], array $headers = [], $ttl = self::TTL_10MIN)
-    {
+    public
```

Comment:
Public on it's own line?
All other functions seem to have this issue, please correct!